torch/testing/_internal/common_methods_invocations.py

```diff
@@ -185,6 +185,8 @@ class OpInfo(object):
     def __init__(self,
                  name,  # the string name of the function
                  *,
+                 ref=None,  # Just a reference
```

Comment:
Let's elaborate in this comment.

"An optional reference function that accepts NumPy arrays. If given, the op will be compared with its reference on each of its sample inputs."

Reply:
Ah, yes, I'm sorry. I understand I need to revise this comment, was just first focusing on the structure of changes. :) But thanks for taking a look and suggesting a better comment, Ruberry. :)

Reply:
Note to self: This is still in my TODOs, I'll work on adding/elaborating the comments once done with the design.